### DISCUSSION TITLE
git cli download speed text

### DISCUSSION THREAD

**Discussion by whitevamp** [git-for-windows/git#4600] [Q&A]
when downloading any size repo, ( really see it with larger ones) the download speed text is incorrect, IE: Filtering content:   5% (25/456), 56.45 GiB | 177.00 KiB/s ( task man- recive: 30.01 Mbps) but my network usage shows at 30meg and if i kill that process, ctrl c my network speed goes down to nothing. or is it doing parallel processing, and not calculating correctly?

**Comment by dscho** [comment]
This is not a Windows-specific issue. Please take it to [the Git mailing list](mailto:git@vger.kernel.org) (send plain-text messages, HTML messages are dropped silently).
